scripts/preprocess/apply_subword.py

```diff
@@ -0,0 +1,157 @@
+import argparse
+import textwrap
+from multiprocessing import Pool
```

Comment:
Should set the serialization format based on our minimum supported version. See https://bugs.python.org/issue28053

Reply:
I'll do it after this PR.